### PR TITLE
IR-250: Allow admin users to access the retrospective app when they are not part of it

### PR DESCRIPTION
**Pull request by naveenkumar1012**

### CODE REVIEW

File: apps/retrospective/services/permission.go [hunk 1]
```diff
@@ -112,3 +137,14 @@ func (service PermissionService) CanAccessRetrospectiveFeedback(sprintID string)
 		Error
 	return err == nil
 }
+
+// IsUserAdmin ...
+func (service PermissionService) IsUserAdmin(userID uint) bool {
+	db := service.DB
+	err := db.Model(&userModels.User{}).
+		Where("users.id=?", userID).
```
Comment (himanshu-garg-165) [inline apps/retrospective/services/permission.go]: Please add spaces around the `=` sign.

File: apps/retrospective/services/retrospective.go [hunk 2]
```diff
@@ -23,7 +23,8 @@ type RetrospectiveService struct {
 }
 
 // List all the Retrospectives of all the teams, given user is a member of.
-func (service RetrospectiveService) List(userID uint, perPageString string, pageString string) (
+func (service RetrospectiveService) List(
+	userID uint, perPageString string, pageString string, isAdmin bool) (
```
Comment (himanshu-garg-165) [inline apps/retrospective/services/retrospective.go]: Do we really need to break the line here? If yes, please ensure it adheres to the standards.

File: apps/retrospective/services/retrospective.go [hunk 3]
```diff
@@ -49,19 +50,31 @@ func (service RetrospectiveService) List(userID uint, perPageString string, page
 	} else {
 		offset = (page - 1) * perPage
 	}
+	if isAdmin {
```
Comment (himanshu-garg-165) [inline apps/retrospective/services/retrospective.go:61]: Please common out the base query-set.

**Review by himanshu-garg-165 — CHANGES_REQUESTED** [review]
Please make sure all the APIs are covered and the admin is able to view/edit everything irrespective of he being in a team or not. Also, ensure that the imports are ordered and grouped properly.

File: apps/retrospective/services/permission.go (hunk 1 above)
Comment (naveenkumar1012) [reply apps/retrospective/services/permission.go]: We have used the same format everywhere in this file. But, I agree we should add spaces around the `=`. I'll add the space everywhere in this file.

File: apps/retrospective/services/retrospective.go (hunk 2 above)
Comment (naveenkumar1012) [reply apps/retrospective/services/retrospective.go]: Are we not following 80 or 100 chars per line? If not, I'll make this in one line.

File: apps/retrospective/services/retrospective.go (hunk 3 above)
Comment (naveenkumar1012) [reply apps/retrospective/services/retrospective.go:61]: Sure.

File: apps/retrospective/services/retrospective.go (hunk 2 above)
Comment (himanshu-garg-165) [reply apps/retrospective/services/retrospective.go]: No, we are not following any such standard as of now (I believe there is no such standard in Go). But if there is some standard in the Go community, then we should follow it.

**Review by himanshu-garg-165 — COMMENTED** [review]
PR looks good. Apparently, there are a few formatting issues, please run `make fmt` to fix them.

**Review by himanshu-garg-165 — APPROVED** [review]

**Pull request merged**